Report.md

```diff
@@ -96,8 +96,43 @@ Design Thinking Approach
 
 
 ## **Agile Methodology - Scrum**
-4. Explain the agile methodology, namely, Scrum you have used to carry out the procedure. In your explanation, ensure that you outline activities from Scrum that you use.
+> 4. Explain the agile methodology, namely, Scrum you have used to carry out the procedure. In your explanation, ensure that you outline activities from Scrum that you use.
```

Comment:
Are we keeping these in the final product?

Reply:
I think we'll remove it